package.json

```diff
@@ -32,7 +32,7 @@
     "e2e-test": "lerna exec \"gulp e2e-test\"",
     "generate-docs": "lerna run --scope=appium generate-docs",
     "generate-schema-declarations": "node ./scripts/generate-schema-declarations.mjs",
-    "postinstall": "lerna bootstrap && lerna exec --parallel \"gulp prepublish\"",
+    "postinstall": "lerna exec --parallel \"gulp prepublish\"",
```

Comment:
omg were we running bootstrap on user installs as well as dev installs?

Reply:
there are no "user installs" of `appium-monorepo`, thankfully. 😉